src/assets/style/global.scss

```diff
@@ -1,3 +1,4 @@
+@import 'reset.scss';
```

Comment:
We already have sanitize.css, which handles this

Reply:
It doesn't properly reset button styling